packages/dcmtk/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE="https://dicom.offis.de/dcmtk"
+TERMUX_PKG_DESCRIPTION="A collection of libraries and applications implementing large parts the DICOM standard"
+TERMUX_PKG_GROUPS="science"
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_LICENSE_FILE="COPYRIGHT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="3.6.7"
+TERMUX_PKG_SRCURL="https://github.com/DCMTK/dcmtk/archive/refs/tags/DCMTK-$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=17705dcdb2047d1266bb4e92dbf4aa6d4967819e8e3e94f39b7df697661b4860
+TERMUX_PKG_DEPENDS="libc++, zlib, libpng, libtiff, libxml2, openssl, libiconv, libjpeg-turbo, libsndfile, libicu"
+TERMUX_PKG_BUILD_IN_SRC=true
```

Comment:
Does it have to be built in source? Have you tried without it?

Reply:
already removed.

Reply:
@licy183 Apparently cmake can't find the airth.h files when building out of source. Any ideas?